bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java

```diff
@@ -47,42 +49,87 @@
     return null;
   }
 
+  private static Configuration setConnectionClass(Configuration config) {
+    Preconditions.checkState(CONNECTION_CLASS != null,
+        "Could not find an appropriate BigtableConnection class");
+    config.set(HConnection.HBASE_CLIENT_CONNECTION_IMPL, CONNECTION_CLASS.getCanonicalName());
+    return config;
+  }
+
   /**
-   * <p>configure.</p>
+   * <p>Create a configuration with HBase resources loaded and Connection Class set.</p>
+   */
+  public static Configuration create() {
+    return setConnectionClass(HBaseConfiguration.create());
```

Comment:
I'm not happy with HBaseConfiguration.create().  It caused us some significant pain in some cases due to the way it looks up XML files in the classpath.